site/source/tutorials/001_static_content/demo/finished/biz-e-corp/tests/unit/widgets/HelloWorld.ts

```diff
@@ -9,7 +9,8 @@ registerSuite({
 		const helloWorld = new HelloWorld();
 
 		const vnode = <VNode> helloWorld.__render__();
-		assert.strictEqual(vnode.vnodeSelector, 'div');
-		assert.equal(vnode.text, 'Hello, Dojo World!');
+		assert.strictEqual(vnode.vnodeSelector, 'h1');
+		assert.equal(vnode.text, 'Biz-E Bodies');
+		assert.equal(vnode.properties.title, 'I am a title!');
```

Comment:
This change is addressed using @vansimke suggested method in d7ae998 -- it is clearer than the one @agubler suggested, which overrides TS rather than asserting that the property actually exists.